deps/build.jl

```diff
@@ -0,0 +1,4 @@
+default_dir = get(ENV, "CONDA_JL_HOME", abspath(joinpath(dirname(@__FILE__), "usr")))
```

Comment:
It should remember the environment variable so that you only need to set it once, and `Pkg.update()` won't "forget" it.  The way I do this in PyCall is to store it in a file.   e.g. do:
```jl
default_dir = get(ENV, "CONDA_JL_HOME", isfile("CONDA_HOME") ? readchomp("CONDA_HOME") : abspath(joinpath(dirname(@__FILE__), "usr")))
```
and then, at the end of the build script (i.e. after a successful build) do:
```jl
open("CONDA_HOME") do io
    println(io, default_dir)
end
```
Alternatively, you could use the `deps.jl` file that you are already writing, i.e. do
```jl
if haskey(ENV, "CONDA_JL_HOME")
    default_dir = ENV["CONDA_JL_HOME"]
elseif isfile("deps.jl")
    include("deps.jl")
else
    default_dir = abspath(dirname(@__FILE__), "usr")
end
```